src/main.rs

```diff
@@ -24,6 +24,14 @@ struct Args {
     /// Approximates bandwidth utilization of cudaMemcpy.
     #[arg(long, exclusive = false)]
     bandwidth_util: bool,
+
+    /// Address for the Prometheus metrics endpoint.
+    #[arg(long, default_value = "0.0.0.0:9091")]
```

Comment:
I would change this to port `9000`. That is generally the default metrics port for applications that export Prometheus metrics. Here I set to `9001` because something else was using that port on my system